api/config/v1alpha1/groupversion_info.go

```diff
@@ -1,7 +1,11 @@
+// Copyright 2022 Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
 // Package v1alpha1 contains API Schema definitions for the config v1alpha1 API group.
 //
-//+kubebuilder:object:generate=true
-//+groupName=config.gateway.envoyproxy.io
+// +kubebuilder:object:generate=true
+// +groupName=config.gateway.envoyproxy.io
```

Comment:
This change is related to gofmt changes in 1.19. Since the project still uses 1.18, please revert.